core/src/main/scala/magnolia1/impl.scala

```diff
@@ -107,29 +140,65 @@ object CaseClassDerivation:
     inline erasedValue[(Labels, Params)] match
       case _: (EmptyTuple, EmptyTuple) =>
         Nil
+      case _: (
```

Comment:
I don't think I understand, why are we matching only empty and 16-tuples? What about all the others?

Reply:
We match 
- empty
- 16
- 4 
- 1

These arities allows to inline tuples up to 27 * 16 = 432 fields:
(in worst case for 431 field: (26 inlines) * (by 16) + (3 inlines) * (by 4) + (3 inlines) * (by 1))

Reply:
Aren't there a few more `inline` calls before the first call to `paramsFromMaps`? I think that the numbers will be slightly different then.
Also I'm wondering if it won't be a bit weird that the set of supported sizes has "holes" i.e. some unsupported sizes are smaller than some supported sizes. That is an extreme example though, so I would say that we should at least handle 0-255 (max number of arguments to a method on the JVM).

Reply:
> Aren't there a few more inline calls before the first call to paramsFromMaps? I think that the numbers will be slightly different then.

Yes, that is true

I have added tests for 254 field (compiler error about platform restriction for 255 fields)
There is no holes in 1-254 range, 254 is a worst case for 16,4,1 arities

The only way to support sizes without holes - arities without holes )
I can do that, but it looks like overkill